stdpopsim/genomes.py

```diff
@@ -386,6 +386,86 @@ def add_dfe(self, intervals, DFE):
         self.dfe_list.append(DFE)
         self.interval_list.append(intervals)
 
+    def add_single_site(
+        self,
+        id,
+        coordinate,
+        selection_coeff=0.0,
+        dominance_coeff=0.5,
+        description=None,
+        long_description=None,
+    ):
+        """
+        Adds a model of a single site mutation at the provided coordinate.
+        The string label of the single site may be referenced by extended
+        events passed to the simulation engine.
+
+        For instance, to simulate a selective sweep,
+
+        .. code-block:: python
+
+            contig.add_single_site(
+                id="sweep_variant",
```

Comment:
```suggestion
                id="hard_sweep",
```

maybe change this name to match above?